pkg/ingester/ingester.go

```diff
@@ -1539,6 +1539,7 @@ func (i *Ingester) MetricsForLabelMatchers(ctx context.Context, req *client.Metr
 	return result, nil
 }
 
+// TODO: Deprecated, remove in v2.14.
 func (i *Ingester) UserStats(ctx context.Context, req *client.UserStatsRequest) (resp *client.UserStatsResponse, err error) {
```

Comment:
We can't remove this. This is used by labels cardinality API endpoint (see `Distributor.labelValuesCardinality()`).